docs/source/notebook.rst

```diff
@@ -314,8 +320,8 @@ shortcut.
 
 A notebook may be downloaded in either a ``.ipynb`` or ``.py`` file from the
```

Comment:
There are now download options for several more formats. Everything apart from .ipynb download works by running the notebook through nbconvert.